### PR TITLE
1966 tooltip

### PR DESCRIPTION
**Pull request by dksmv**

### CODE REVIEW

**Comment by mf-code** [comment]
[5886] - 78fQTLByKYPH

**Comment by mf-code** [comment]
[5886] - Tpn9aiZGLPeB

**Comment by mf-code** [comment]
[5886] - cFpASwpeW7el

**Comment by mf-code** [comment]
[5886] - AzxR9U7pLv1Q

**Comment by mf-code** [comment]
[5886] - oNTJ7TNYjng2

**Comment by mf-code** [comment]
[5886] - OCGH119RlqN8

**Comment by mf-code** [comment]
[5886] - cyVyUKaD4Q4I

**Comment by mf-code** [comment]
[5886] - gSyvStXL8JA9

**Comment by mf-code** [comment]
[5886] - DP8BRxdHadP8

**Comment by mf-code** [comment]
[5886] - TSf088TCujlB

**Comment by mf-code** [comment]
[5886] - C8GjhOwVS6Og

**Comment by mf-code** [comment]
[5886] - 3LQK81npU1v2

**Comment by mf-code** [comment]
[5886] - yiE9wFQytnoO

**Comment by mf-code** [comment]
[5886] - g5ejFf9tXcpp

**Comment by mf-code** [comment]
[5886] - openjdk8 - Allure report on Netlify: https://5f3aab72fb8f784674f53ae5--musing-swartz-6fc060.netlify.app

**Comment by mf-code** [comment]
[5886] - openjdk10 - Allure report on Netlify: https://5f3aac66a068ce35be28be37--musing-swartz-6fc060.netlify.app

File: jdi-light-angular/src/main/java/com/epam/jdi/light/angular/asserts/TooltipAssert.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+import com.epam.jdi.light.elements.base.UIBaseElement;
+import org.hamcrest.Matchers;
+
+import static com.epam.jdi.light.asserts.core.SoftAssert.jdiAssert;
+
+public class TooltipAssert extends UIAssert<TooltipAssert, Tooltip> {
+
+    @JDIAction("Assert that '{name}' has '{0}' text")
+    public TooltipAssert text(String text) {
+        jdiAssert(element.text(), Matchers.is(text));
```
Comment (Vtablos) [inline jdi-light-angular/src/main/java/com/epam/jdi/light/angular/asserts/TooltipAssert.java:15]: rename assert assertText

File: jdi-light-angular/src/main/java/com/epam/jdi/light/angular/asserts/TooltipAssert.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+import com.epam.jdi.light.common.JDIAction;
+import com.epam.jdi.light.elements.base.UIBaseElement;
+import org.hamcrest.Matchers;
+
+import static com.epam.jdi.light.asserts.core.SoftAssert.jdiAssert;
+
+public class TooltipAssert extends UIAssert<TooltipAssert, Tooltip> {
+
+    @JDIAction("Assert that '{name}' has '{0}' text")
+    public TooltipAssert text(String text) {
```
Comment (Vtablos) [inline jdi-light-angular/src/main/java/com/epam/jdi/light/angular/asserts/TooltipAssert.java:14]: rename assert assertTooltipText

File: jdi-light-angular/src/main/java/com/epam/jdi/light/angular/asserts/TooltipAssert.java [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+        jdiAssert(element.text(), Matchers.is(text));
+        return this;
+    }
+
+    @JDIAction("Assert that '{name}' has '{0}' color")
+    public TooltipAssert color(String color) {
```
Comment (Vtablos) [inline jdi-light-angular/src/main/java/com/epam/jdi/light/angular/asserts/TooltipAssert.java:20]: rename assert assertTooltipColor

File: jdi-light-angular/src/main/java/com/epam/jdi/light/angular/asserts/TooltipAssert.java [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+        jdiAssert(element.color(), Matchers.is(color));
+        return this;
+    }
+
+    @JDIAction("Assert that '{name}' has '{0}' position relative to '{1}' ")
+    public TooltipAssert position(Tooltip.Position position, UIBaseElement<?> e) {
```
Comment (Vtablos) [inline jdi-light-angular/src/main/java/com/epam/jdi/light/angular/asserts/TooltipAssert.java:26]: rename assert assertTooltipPosition

**Comment by mf-code** [comment]
[5900] - 5aV4w8CL0PaV

**Comment by mf-code** [comment]
[5900] - YXihUUZr1uLQ

**Comment by mf-code** [comment]
[5900] -

**Comment by mf-code** [comment]
[5900] - ZRH6NBriI0fO

**Comment by mf-code** [comment]
[5900] - QzNNs7y13fEf

**Comment by mf-code** [comment]
[5900] - Iv0szofUvWqK

**Comment by mf-code** [comment]
[5900] - XNwsuVGRQkaP

**Comment by mf-code** [comment]
[5900] - j5kiUenmKcBQ

**Comment by mf-code** [comment]
[5900] - aFioDz0mOEl7

**Comment by mf-code** [comment]
[5900] - vSBCoHY3Kl9U

**Comment by mf-code** [comment]
[5900] - Ye8omfSO208y

**Comment by mf-code** [comment]
[5900] - nArTBR00NNXD

**Comment by mf-code** [comment]
[5901] - 7SPMQa491kVK

**Comment by mf-code** [comment]
[5901] - 6WgdHDa9Aagk

**Comment by mf-code** [comment]
[5901] - ISLzcItrdUyS

**Comment by mf-code** [comment]
[5901] - 9r4nANvRzAVf

**Comment by mf-code** [comment]
[5900] - XT4mIafWEXSY

**Comment by mf-code** [comment]
[5901] - 9h1aP321styw

**Comment by mf-code** [comment]
[5901] - bs1ymM0WApBW

**Comment by mf-code** [comment]
[5901] - YzFggGHh3DWO

**Comment by mf-code** [comment]
[5901] - nOFzXDkCaxEn

**Comment by mf-code** [comment]
[5901] - LEzvHAiyGyS9

**Comment by mf-code** [comment]
[5901] - 7VUu9plReKa7

**Comment by mf-code** [comment]
[5901] - ZUMuB7PWLMaP

**Comment by mf-code** [comment]
[5901] - K55EsZedzfft

**Comment by mf-code** [comment]
[5910] - f4c5cb88cSuL

**Comment by mf-code** [comment]
[5910] - Czv319JubTqQ

**Comment by mf-code** [comment]
[5910] - aQHM4ucDD1k0

**Comment by mf-code** [comment]
[5910] - SNBM7AQWkh71

**Comment by mf-code** [comment]
[5910] - 1tKCNiyVhAKJ

**Comment by mf-code** [comment]
[5910] - XL4EoCxHtJ7m

**Comment by mf-code** [comment]
[5910] - QBgfw0WkzGe2

**Comment by mf-code** [comment]
[5910] - jFWeVN00WORA

**Comment by mf-code** [comment]
[5910] - bzBTQQSQhpkc

**Comment by mf-code** [comment]
[5910] - 56BurRSyP72z

**Comment by mf-code** [comment]
[5910] - Ll92LJWk37Wo

**Comment by mf-code** [comment]
[5910] - Lky2pc73b70o

**Comment by mf-code** [comment]
[5910] - tq3Tyn3Xuxce

**Comment by mf-code** [comment]
[5910] - WhgN3UJs0gxw

**Comment by mf-code** [comment]
[5910] - LJ1K6cXxdwPu

**Comment by mf-code** [comment]
[5910] - 4EIcwQrwQVow

**Comment by mf-code** [comment]
[5910] - OuwykBPih5cn

**Comment by mf-code** [comment]
[5910] - TIsQCzSLg0hD

**Comment by mf-code** [comment]
[5910] - RvCsvAkn2bSi

**Comment by mf-code** [comment]
[5910] - ADAh8F4CEf7s

**Comment by mf-code** [comment]
[5910] - 0DzX6w6v5U7N

**Comment by mf-code** [comment]
[5910] - NWnGST28vHDm

**Comment by mf-code** [comment]
[5910] - vi2CpvwLDQFP

**Comment by mf-code** [comment]
[5910] - vx3JeDWiw28R

**Comment by mf-code** [comment]
[5910] - smLdZRL2yl6m

**Comment by mf-code** [comment]
[5910] - mlcUKaE43iyI

**Comment by mf-code** [comment]
[5910] - E4YRBH90RpDo

**Comment by mf-code** [comment]
[5910] - 9qGuVJ76c08E

**Comment by mf-code** [comment]
[5910] - orZozJcSxqfJ

**Comment by mf-code** [comment]
[5910] - 8xX0J0RxD05r

**Comment by mf-code** [comment]
[5910] - hFkdp4lNROp4

**Comment by mf-code** [comment]
[5910] - G81N8f4e6nj9

**Comment by mf-code** [comment]
[5910] - cdeJZv9SI0gF

**Comment by mf-code** [comment]
[5910] - LBYNUQcUVWFz

**Comment by mf-code** [comment]
[5910] - 3ClgrqYFv9rg

**Comment by mf-code** [comment]
[5910] - mOHM2T1c2TQS

**Comment by mf-code** [comment]
[5910] - 0F6NjkCzrIvr

**Comment by mf-code** [comment]
[5910] - H26Qv538qJHW

**Comment by mf-code** [comment]
[5931] - yYomBL83qu6g

**Comment by mf-code** [comment]
[5931] - iAzuvNTWu0ee

**Comment by mf-code** [comment]
[5931] - STCHuPVMutHO

**Comment by mf-code** [comment]
[5931] - P32Jl1lYdkJa

**Comment by mf-code** [comment]
[5931] - YfTUHJBg9ggK

**Comment by mf-code** [comment]
[5931] - GDz0IT3UgKCy

**Comment by mf-code** [comment]
[5931] - Uh2Yzpdb2PpR

**Comment by mf-code** [comment]
[5931] - zi4QEeaDHsB3

**Comment by mf-code** [comment]
[5931] - Oi2bu7WUYdm1

**Comment by mf-code** [comment]
[5931] - RVDTWW59f8EU

**Comment by mf-code** [comment]
[5931] - 3zWnvFpGEN4U

**Comment by mf-code** [comment]
[5931] - 64OE7cBtxBn8

**Pull request closed without merging**